### PR TITLE
XF-53-no-else-return

### PR DESCRIPTION
**Pull request by Bartosz-L**
Changed unnecessary if else statements accordingly to no-else-return rule from eslint

### CODE REVIEW

**Review by latata — APPROVED** [review]

File: app/components/article-content.js [hunk 1]
```diff
@@ -243,13 +243,13 @@ export default Component.extend(
         // as the 'figcaption' is contained in the 'figure' element which has the '.article-media-thumbnail' class.
         if (element.closest('.portable-infobox')) {
           return 'portable-infobox-link';
-        } else if (element.closest('.context-link')) {
+        } if (element.closest('.context-link')) {
```
Comment (hakubo) [inline app/components/article-content.js]: if in new line

File: app/mixins/wiki-page-handler.js [hunk 2]
```diff
@@ -126,21 +125,20 @@ export default Mixin.create({
...
-    } else {
-      const wikiPageData = shoebox.retrieve('wikiPage');
-      const wikiPageError = shoebox.retrieve('wikiPageError');
-
-      // There is no way to remove stuff from shoebox, so ignore it on the consecutive page views
-      if (wikiPageError && isInitialPageView) {
-        throw wikiPageError;
-      }
+    }
+    const wikiPageData = shoebox.retrieve('wikiPage');
```
Comment (hakubo) [inline app/mixins/wiki-page-handler.js]: one \n before const

File: app/components/article-content.js (hunk 1 above)
Comment (Bartosz-L) [reply app/components/article-content.js]: done

File: app/models/user.js [hunk 3]
```diff
@@ -34,22 +34,19 @@ export default EmberObject.extend({
...
-        } else {
-          this.logger.error('Helios connection error: ', response);
-        }
-
-        return null;
       }
+      if (response.status === 401) {
+        this.logger.info('Token not authorized by Helios');
+      }
+      this.logger.error('Helios connection error: ', response);
```
Comment (hakubo) [inline app/models/user.js]: now this will be logged always
and used to be logged only when response.status !== 401

**Review by hakubo — APPROVED** [review]

**Pull request merged**